components/core/Server.py

```diff
@@ -46,3 +42,5 @@ def serve_ui(path):
 def serve_ui1():
     return send_file(os.path.dirname(os.path.realpath(__file__))+"/ui/index.html")
 
+# Import routes
+from routes import *
```

Comment:
why is this at the bottom?

Reply:
Circular imports

We need the rest of the code defined before the routes can use them